docs/topics/api/abuse.rst

```diff
@@ -0,0 +1,59 @@
+=============
+Abuse Reports
+=============
+
+The following API endpoint covers abuse reporting
+
+---------------------------------
+Submitting an add-on abuse report
+---------------------------------
+
+.. _`addonabusereport-create`:
+
+The following API endpoint allows an abuse report to be submitted for an Add-on,
+either listed on https://addons.mozilla.org or not.
+Authentication is not required, but is recommended so reports can be responded
+to if nessecary.
+
+.. http:post:: /api/v3/abuse/reportaddon/
```

Comment:
`/api/v3/abuse/report-addon/` or `/api/v3/abuse/report/addon/` would be easier to read/cleaner... and I think using the latter makes more sense as a report API then has subresources (`addon` and `user`) and the API should separate them out.